scripts/setOracleSigningAddress.ts

```diff
@@ -0,0 +1,31 @@
+import { ethers } from "hardhat";
+import { getConfig } from "../helpers/helperUtils";
+import { CONTRACT_NAMES } from "../helpers/constants";
+
+async function main() {
+  const oracleSigningAddress = "0xf0Ae6D287aF14f180E1FAfe3D2CB62537D7b1A82"; // production signing address
+
```

Comment:
We still have the same address in constants, so better to export it as ORACLE_SIGNING_ADDRESS_PRODUCTION from there.